### ISSUE TITLE
Default when editing a commit message

### ISSUE THREAD

**Issue by flimzy** [stevan/p5-App-Critique#20]
What would you like to edit? (f)ile, (c)ommit message: c
    ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
    Please write a commit message: : 

It might be nice to show the original commit message as the default.  Or better, perhaps, open the commit message in an editor, with the default in place (i.e. `git commit -e -F`)

**Comment by stevan** [comment]
Yup, I have run into this as a pain point myself, this is absolutely needed, and since we already have you using an editor, ... ;)
